### ISSUE TITLE
Pull Remote source for HLS only when there are clients connected

### ISSUE THREAD

**Issue by AlexMeteor** [arut/nginx-rtmp-module#729]
On the rtmp (flash) side, it is easy to pull a remote source by using "exec_pull" to execute ffmpeg on a remote video source (e.g rtsp video streaming) since rtmp client does trigger events so exec_pull got executed. 

I understand from "FAQ" that , "Currently HLS clients do not trigger any events. " therefore , there's no way to use similar way to generate a HLS source using exec_pull but it sugguested to use exec_static. However, in my case,  always pulling the remote source are very expensive. Is there any way we can achieve similar functionality just like the rtmp server can do? such as using a http control API call to start/stop both rtmp and hls ? instead of doing it by rtmp events triggering?

Thank you very much in advance for any helps

**Comment by ptowns188** [comment]
would love to see this aswell

**Comment by yonailo** [comment]
Yes please such a feature is really very important to save bandwidth, +1

**Comment by qerim** [comment]
+1
